wallet/src/de/schildbach/wallet/ui/invite/InviteAlreadyClaimedDialog.kt

```diff
@@ -19,59 +19,89 @@ package de.schildbach.wallet.ui.invite
 import android.content.Context
 import android.os.Bundle
 import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.core.text.HtmlCompat
+import androidx.core.view.isVisible
 import de.schildbach.wallet.data.InvitationLinkData
 import de.schildbach.wallet.database.entity.DashPayProfile
 import de.schildbach.wallet_test.R
-import de.schildbach.wallet_test.databinding.InvitationAlreadyClaimedViewBinding
-import org.dash.wallet.common.ui.FancyAlertDialog
 import org.dash.wallet.common.ui.avatar.ProfilePictureDisplay
+import org.dash.wallet.common.ui.dialogs.AdaptiveDialog
 
-open class InviteAlreadyClaimedDialog : FancyAlertDialog() {
-
+open class InviteAlreadyClaimedDialog : AdaptiveDialog(R.layout.invitation_already_claimed_view) {
     companion object {
-
         private const val EXTRA_PROFILE = "profile"
         private const val EXTRA_INVITE = "invite"
 
-        fun newInstance(context: Context, profile: DashPayProfile): FancyAlertDialog {
+        fun newInstance(context: Context, profile: DashPayProfile): AdaptiveDialog {
             return newInstance(context, profile.nameLabel).apply {
                 arguments!!.putParcelable(EXTRA_PROFILE, profile)
             }
         }
 
         @JvmStatic
-        fun newInstance(context: Context, invite: InvitationLinkData): FancyAlertDialog {
+        fun newInstance(context: Context, invite: InvitationLinkData): AdaptiveDialog {
             return newInstance(context, invite.displayName).apply {
                 requireArguments().putParcelable(EXTRA_INVITE, invite)
             }
         }
 
-        fun newInstance(context: Context, nameLabel: String): FancyAlertDialog {
+        fun newInstance(context: Context, nameLabel: String): AdaptiveDialog {
```

Comment:
Nice. I hope we can get rid of `FancyAlertDialog` some day :)

Reply:
Most references are now gone.